x-pack/plugins/index_management/public/application/components/mappings_editor/components/configuration_form/source_field_section/source_field_section.tsx

```diff
@@ -27,7 +77,7 @@ export const SourceFieldSection = () => {
       <p>
         <FormattedMessage
           id="xpack.idxMgmt.mappingsEditor.disabledSourceFieldCallOutDescription1"
-          defaultMessage="Disabling {source} lowers storage overhead within the index, but this comes at a cost. It also disables important features, such as the ability to reindex or debug queries by viewing the original document."
+          defaultMessage="Disabling {source} is not recommended. If storage overhead is a concer, consider using synthetic {source} instead. Disabling {source} will disable important features, such as the ability to reindex or debug queries by viewing the original document."
```

Comment:
```suggestion
          defaultMessage="Disabling {source} is not recommended. If storage overhead is a concern, consider using synthetic {source} instead. Disabling {source} will disable important features, such as the ability to reindex or debug queries by viewing the original document."
```

Reply:
Thanks @florent-leborgne! Fix the typo with [aa2b19b](https://github.com/elastic/kibana/pull/199854/commits/aa2b19b371d74756506063f2716ed8f3f8353046).